lib/populate.py

```diff
@@ -100,12 +100,9 @@ def safe_request(cmd, *args, **kwargs):
 
 
 def get_streams(client):
-    # In the future, we may want to change this to
-    # include_web_public=True, for organizations that might want to
-    # use the upcoming web_public flag; but at the very least we
-    # should only include public streams.
     response = safe_request(
-        client.get_streams, include_public=True, include_subscribed=False
+        client.get_streams, include_public=True, include_subscribed=False,
+        include_web_public=True
     )
```

Comment:
I think ideally we'd make this more of an option that can be controlled by users -- with a few settings in the configuration file for whether to include all web-public streams, all public streams, and maybe an option to list a specific set of streams.

Reply:
The public streams stuff can be done in a separate PR. I made #93 (which is finished, and currently I'm testing on julialang.zulipchat.com's web-public streams), so this PR can be closed.